pkg/features/kube_features.go

```diff
@@ -1033,7 +1033,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	DisableCloudProviders: {Default: true, PreRelease: featuregate.Beta},
```

Comment:
I think we also agreed to graduate this feature gate to GA in 1.31. cc @cheftako @elmiko

Reply:
yes, that was the plan in the enhancement

Reply:
doing this now.